interop-qa-tests/src/test/java/it/pagopa/pn/interop/cucumber/steps/agreement/AgreementCommonSteps.java

```diff
@@ -51,8 +47,8 @@ public void tenantHasAlreadyCreatedAndPublishedEService(String tenantType, int t
         for (int i = 0; i < totalEservices; i++) {
             // Create e-service and descriptor
             int randomInt = ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE);
-            int TEST_SEED = ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE);
-            String eserviceName = String.format("eservice-%d-%d-%d", i, TEST_SEED, randomInt);
+            int testSeed = ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE);
+            String eserviceName = String.format("eservice-%d-%d-%d", i, testSeed, randomInt);
```

Comment:
è possibile rimuovere la generazione di un nuovo valore per testSeed e riutilizzare quello già contenuto in `sharedStepsContext`
```suggestion
            String eserviceName = String.format("eservice-%d-%d-%d", i, sharedStepsContext.getTestSeed(), randomInt);
```